src/notifications/testers.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * List of push notification testing accounts.
+ * When logged in as one of these accounts it will email the
+ * push notification token to the listed email. This is used
+ * for testing push notifications before they are sent to everyone.
+ */
+export const pushTesters = [
```

Comment:
Thanks for the feedback, I broke down the large class so it should be a little smaller now. As far as the user classifications those should be running asynchronously and should not block the user, however, if you have a user that has a lot of classifications that I could test with, it would help me confirm. Also I had a discussion with @lcjohnso  about the classifications and I believe it was agreed to not use the ERAS endpoint at this time (Cliff, please correct me if I misunderstood this). I get the classifications anytime the project list on the home screen is refreshed, so I want to make sure I'm avoiding any potential performance issues.

You can use a zootester login and/or I can add you to the tester list if you want to send me your username and the email you would want the test token sent to. (We can jump on a call if needed and I can walk your through everything and make sure you have what you need to test/PR).